README.md

```diff
@@ -71,26 +90,23 @@ $> botonic serve
 $> botonic deploy
 ```
 
-Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started.html) is a step-by-step guide to start building high quality conversational apps.
-
-[See the docs for more information](https://botonic.io/docs.html).
+Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started) is a step-by-step guide to start building high quality conversational apps.
```

Comment:
Check out the [Getting Started Tutorial](https://botonic.io/docs/getting-started): a step-by-step guide to start building high quality conversational apps.